frontend/package.json

```diff
@@ -45,7 +45,7 @@
     "react-redux": "^7.2.1",
     "react-router": "^5.2.0",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "^4.0.3",
+    "react-scripts": "^5.0.0",
```

Comment:
Is this related to the vulnerability? There are definitely breaking changes here.

Reply:
https://github.com/facebook/create-react-app/releases/tag/v5.0.0

Reply:
Apologies, that got slipped in by mistake! I was investigating something the snyk bot was doing earlier. Let me revert that file.